### ISSUE TITLE
Help with my sync

### ISSUE THREAD

**Issue by dinhanhuy** [ethereum/mist#4084]
PLease help me, how can i use eth wallet? it always show this 
![hd](https://user-images.githubusercontent.com/41427677/44784503-16d21900-abb8-11e8-8c8e-09d30f38e54a.PNG)

**Comment by jfgreen01** [comment]
yep, even with new client still can't connect; this is why ethereum is falling.

**Comment by cryptojunky** [comment]
Same issue here.  2 hours since last block and stuck on connecting. Tried running geth --light still gets stuck.

**Comment by jfgreen01** [comment]
It's sad I am coming back here and try to help instead of the dev team.

If two hours it's not going to connect; Are you running the latest version?  I had success in the passed few days; your chainstate data is probably corrupt as you were going back and forth trying to sync blocks multiple / various ways.

Delete your geth/chaindata/ and geth/lightchaindata - these can be found under folder users/yourprofile/appdata/roaming/ethereum/geth/

try to run again latest version.

**Comment by cryptojunky** [comment]
I tried that and deleting all but keystore files. It syncs one or two blocks than doesn't sync for hours. Going to try rolling back a version. Thanks anyway.

**Comment by cryptojunky** [comment]
I went back to 0.9.3 which worked. I start geth first using cmd promt and geth --light. Than open ethereum. works fine that way.

**Comment by jfgreen01** [comment]
did you try 10.0.0? I wouldn't go back too far as there could be security and or vulnerabilities in the software; or if you really need to make a transaction the dev team told us to use MEW HAHHA.

**Comment by jesobreira** [comment]
Was having the same issue. Full of E_TIMEDOUT on the logs. Got it working by installing Geth manually ([instructs for Ubuntu](https://github.com/ethereum/go-ethereum/wiki/Installation-Instructions-for-Ubuntu)).

**Comment by hydra** [comment]
Same, new version since 0.10.0 doesn't sync, old version 0.9.2 syncs just fine.

* 0.10.0 no sync
* 0.11.0 no sync
* 0.11.1 no sync

Latest ones only seem to connect to a few peers then it says 'Remote' instead of a number, shows a block count and a clock that ticks upwards.  The block count increases.  The download progress bar is never displayed.

With 0.9.2 the download progress bar appears after about 10-20 seconds.
